item-spec/item-spec.md

```diff
@@ -132,31 +132,38 @@ The following types are commonly for assets and are used as key for the Asset Ob
 | --------- | -------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
 | thumbnail | STRONGLY RECOMMENDED. A downsampled image of the core asset, for direct display online in a web page or interactive search application. Even assets that are less easily translated in to a visual image should provide some visual representation, that users can browse. For example a SAR asset can render an elevation mask or hillshade for display. If at all possible it should be included for a better user experience in searching data. |
 
-#### MIME types
+#### Media Types
 
-[Registered](https://www.iana.org/assignments/media-types/media-types.xhtml) MIME types are
-preferred when possible. In cases where custom, vendor specific MIME types are necessary they can be
-used with the `vnd.` prefix for the type. The MIME type can be used by STAC browsers to better
-determine what is relevant to render and display to users searching and browsing the catalog. In
-some cases, selection of a mime type can be ambiguous. For instance, many STAC items have sidecar
-metadata files associated with the core asset (`.tfw`, Landsat 8 MTL files, etc.) that should use
-mime types appropriate for the file - for instance, if it is a plain text file then `text/plain`
-would be appropriate, `text/xml` would be appropriate for cases the metadata is in XML format.
+The media type of an Item can be used by STAC browsers to better determine what to render and display 
+to users searching and browsing the catalog. 
 
-A set of commonly used MIME types for STAC items include the following:
+[Registered](https://www.iana.org/assignments/media-types/media-types.xhtml) Media Types are
+preferred. In cases where custom vendor-specific media types are necessary, they should
+use the `vnd.` prefix. 
 
-| MIME type                      | Description                                                                             |
+STAC Items that have sidecar metadata files associated with a data asset (e.g, `.tfw`, Landsat 8 MTL files) 
+should use media types appropriate for the the metadata file.  For example, if it is a plain text file, then `text/plain`
+would be appropriate; if it is an XML, then `text/xml` is appropriate.
```

Comment:
Should we somehow mention that `application/xml` can be used as synonym for `text/xml` by applications? See [RFC 7303](https://www.ietf.org/rfc/rfc7303.txt) for details.

Reply:
Added a reference to `application/xml`